ted_sws/core/entrypoints/api/main.py

```diff
@@ -0,0 +1,124 @@
+import hashlib
+import logging
+import subprocess
+import uuid
+from enum import Enum
+from typing import Optional, Union, Tuple
+
+import click
+from fastapi import FastAPI
+
+from ted_sws.event_manager.adapters.logger import Logger, LOG_INFO_LEVEL
+from ted_sws.event_manager.domain.message_bus import message_bus
+from ted_sws.event_manager.model.message import Log
+
+
+class ResponseType(Enum):
+    JSON = "json"
+    RAW = "raw"
+
+
+class UUIDInputProcessType(Enum):
+    MD5 = "md5"
+    RAW = "raw"
+
+
+class UUIDNamespaceType(Enum):
+    DNS = "DNS"
+    URL = "URL"
+    OID = "OID"
+    X500 = "X500"
+
+
+API_SERVER_LOG_LEVELS: Tuple = ('critical', 'error', 'warning', 'info', 'debug', 'trace')
```

Comment:
surely these constants are already available elsewhere and we do not need re-defining them here.